packages/@aws-cdk/aws-cloud9-alpha/lib/environment.ts

```diff
@@ -257,6 +257,26 @@ export class Owner {
     return { ownerArn: user.userArn };
   }
 
+  /**
+   * Make an IAM assumed role the environment owner
+   *
+   * @param accountId The AccountId of the target account
+   * @param roleName The name of the assumed role
+   */
+  public static assumedRole(accountId: string, roleName: string): Owner {
```

Comment:
Can this be named `assumeRole`?

Reply:
This maps to an IAM identifier called assume**d**-role, so I think this is the best name for the function